Makefile

```diff
@@ -40,7 +39,7 @@ test-acceptance:
 generate:
 	go generate ./...
 
-lint:	fmt vet tidy
+lint:	prepare_version fmt vet tidy
```

Comment:
If `lint` depends on `prepare_version`, the changes in lines 11,14,17,20, and 23 are superfluous.

Reply:
Yes, I realized that yesterday after fixing linting, but I thought it is confusing if linting is required for building to be successful. Running the step twice should not heart so I left it but I can remove it if the dependency is not an issue.

Reply:
> I  thought it is confusing if linting is required for building to be successful.

Based on how the Makefile is written we are requiring lint to be successful before we are even starting to build.  That is IMHO a good thing, i.e. as `vet` tells us about issues we might encounter first. Not just plain syntax errors, also more general things, like outdated constructions. I do not see the confusion.

Having the `prepare_version` required for lint simply makes that even more clear.

... So, I would go with @kkaempf here.